dotcom-rendering/src/web/components/Caption.tsx

```diff
@@ -141,6 +149,7 @@ export const Caption = ({
 	shouldLimitWidth = false,
 	isOverlayed,
 	isLeftCol,
+	isVideo = false,
```

Comment:
Could we change this to `mediaType`? CAPI already has this a type and it's defined as a global in DCR's index.d.ts file:

```ts
type MediaType = 'Video' | 'Audio' | 'Gallery';
```

Reply:
Good suggestion, thanks Oliver